Clocks/FMI3.xml

```diff
@@ -7,17 +7,17 @@
  </LogCategories>
  <DefaultExperiment stopTime="10" stepSize="1"/>
  <ModelVariables>
-  <Clock name="inClock1" valueReference="1001" causality="input" interval="constant" intervalDecimal="1.0" priority="0"/>
-  <Clock name="inClock2" valueReference="1002" causality="input" interval="triggered" priority="1"/>
-  <Clock name="inClock3" valueReference="1003" causality="input" interval="countdown" priority="2"/>
-  <Clock name="outClock" valueReference="1005" causality="output" interval="triggered" priority="2"/>
-  <Int32 name="inClock1Ticks" valueReference="2001" causality="output"/>
-  <Int32 name="inClock2Ticks" valueReference="2002" causality="output"/>
+  <Clock name="inClock1" valueReference="1001" start="0" causality="input" interval="constant" intervalDecimal="1.0" priority="0"/>
```

Comment:
I think start values for clocks are pointless and confusing. But it is correct according to the current standard.
I will open a sperate ticket about it